honeybee_radiance_command/dcglare.py

```diff
@@ -0,0 +1,186 @@
+"""dcglare command."""
+
+from .options.dcglare import DcglareOptions
+from ._command import Command
+import honeybee_radiance_command._exception as exceptions
+import honeybee_radiance_command._typing as typing
+
+
+class Dcglare(Command):
+    """Dcglare command.
+
+    Dcglare generates daylight glare probability (DGP) predictions for multiple points in
+    a space under a variety of daylit conditions. Usually, it is used to produce hourly
+    DGP values for an entire year, or if the -l option is provided, it calculates glare
+    autonomy based on an annual occupancy schedule.
+
+    As input, dcglare requires daylight coefficient matrices relating the illuminance at
+    each view point to the brightness of each sky patch. Two such matrices are required.
+    The first, DCdirect, consists of direct views to the sky only and is calculated by
+    rcontrib using a single ambient bounce. The second, DCtotal, includes the total
+    direct and diffuse contribution of each sky patch. The latter can be calculated
+    directly by rcontrib as in the two-phase method, or internally as in the three-phase
+    method if given view, BSDF, and daylight matrices. In this respect, dcglare is
+    similar to dctimestep except that it calculates DGP instead of irradiance. The final
+    input is the sky contribution matrix, usually computed by gendaymtx, which may be
+    passed on the standard input. For efficiency, matrices stored in files can be
+    represented as binary float data if machine byte-order is not an issue.
+
+    Args:
+        options: Command options. It will be set to Radiance default values
+            if unspecified.
+        output: File path to the output file (Default: None).
+        dcdirect: File path to the direct contribution file (Default: None).
+        dctotal: File path to the total (direct and diffuse) contribution file
+            (Default: None).
+        sky_matrix: File path to the sky contribution file (Default: None).
+
+    Properties:
+        * options
+        * output
+        * dcdirect
+        * dctotal
```

Comment:
IMO, it will be more readable to use `dc_direct` and `dc_total`.

```suggestion
        * dc_direct
        * dc_total
```